### PR TITLE
Adjusted some events

### PR DESCRIPTION
**Pull request by EdNoepel**
- Included `lup` in `RemoveQuoteToken` and made naming consistent in `Borrow` and `Repay` events
- Implement `UpdateInterestRate` event

### CODE REVIEW

File: tests/test_interest_rate.py [hunk 1]
```diff
@@ -39,12 +39,16 @@ def test_update_interest_rate(
 
     tx = mkr_dai_pool.updateInterestRate({"from": lender})
     assert tx.status.value == 1
-    compare_first_16_digits(
-        Decimal(mkr_dai_pool.previousRate()), Decimal(87500000059455860)
+    assert compare_first_16_digits(
+        Decimal(mkr_dai_pool.previousRate()), Decimal(87500000000000000)
```
Comment (EdNoepel) [inline tests/test_interest_rate.py]: This test wasn't doing anything; it was merely running a method which returned `False`, which wasn't helpful.

**Review by grandizzy — COMMENTED** [review]
LGTM, minor comments
- do we also want to add `indexed` events params with this PR?
- pls configure code format / lint on file save

**Comment by EdNoepel** [comment]
> LGTM, minor comments
> 
> * do we also want to add `indexed` events params with this PR?
> * pls configure code format / lint on file save

Latest commits resolve test issue, lint the first two events of the pool contract, and set `indexed` on appropriate event parameters.

**Review by grandizzy — APPROVED** [review]
LGTM

**Pull request merged**